modules/phase_field/include/userobjects/DiscreteNucleationInserter.h

```diff
@@ -41,6 +44,12 @@ class DiscreteNucleationInserter : public DiscreteNucleationInserterBase
   /// the local nucleus list of nuclei centered in the domain of the current processor
   NucleusList & _local_nucleus_list;
 
-  /// total nucleation rate
+  /// Total nucleation rate. For time-dependent statistics, this is probability rate density, for time-independent statistics, it is probability density
```

Comment:
this line is too long (clang-format)

Reply:
How do I make this a multi-line comment that will still get picked up?

Reply:
fixed